src/applications/virtual-agent/tests/components/webchat/Webchat.unit.spec.js

```diff
@@ -3,7 +3,7 @@ import sinon from 'sinon';
 import { setMicrophoneMessage } from '../../../components/webchat/WebChat';
 
 describe('WebChat.jsx', () => {
-  describe('setMicrophoneMessage', () => {
+  describe.skip('setMicrophoneMessage', () => {
```

Comment:
We ran this locally and it passes as is. Can you explain the reason for this change?

Reply:
@szarrougtw When you ran this locally, did you run only the `virtual-agent` test suite or did you run the entire suite? The reason why I had this change is because when running the entire testing suite, this particular unit test exeeds the 10s timeout limit and throws an error.

![Screenshot 2024-03-11 at 2 50 54 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/67602137/0c7f378a-d374-4e21-8bcb-77ba7a7f02b0)

Reply:
yeah, i had only run ours. looks like running everything does indeed timeout, so weird! we'll fix this separately, thank you! approved for virtual agent